core/src/main/java/org/kiwix/kiwixmobile/core/data/local/entity/RecentSearchSpec.kt

```diff
@@ -15,16 +15,19 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.core.data.local.entity;
+package org.kiwix.kiwixmobile.core.data.local.entity
 
-import com.yahoo.squidb.annotations.TableModelSpec;
-import com.yahoo.squidb.annotations.tables.constraints.NotNull;
+import com.yahoo.squidb.annotations.TableModelSpec
+import com.yahoo.squidb.annotations.tables.constraints.NotNull
 
 /**
  * Squidb spec for recent searches.
  */
 @TableModelSpec(className = "RecentSearch", tableName = "recentSearches")
-public class RecentSearchSpec {
-  @NotNull public String searchString;
-  @NotNull String zimID;
+class RecentSearchSpec {
+  @NotNull
+  lateinit var searchString: String
+
+  @NotNull
+  lateinit var zimID: String
```

Comment:
Why it should be late initializer property?

Reply:
hi @gouri-panda ,
we should make this variables null by default?

Reply:
>Why it should be late initializer property?

That was a question. Do you think nullable is a good idea with `zimId`?

Reply:
Hi @gouri-panda ,

we are using ```@NonNull``` annotation of ```com.yahoo.squidb.annotations.tables.constraints.NotNull``` on both ```searchString``` and ```zimId``` so on runtime these variables are treat as non null.

Reply:
hi @gouri-panda , @kelson42 
this PR is ready for re-review.